src/contexts/RoomContext.js

```diff
@@ -0,0 +1,25 @@
+import React, {createContext, useState, useEffect} from "react";
+
+
+export const RoomContext = createContext();
+
+const RoomContextProvider = (props) => {
+
+      const [roomName, setRoomName] = useState(() => {
```

Comment:
i was thinking things like `all_players` would be in a room context and individual player things like `current_cards` could be in player context.